data-client/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="org.wikipedia.dataclient">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Refer to this [comment](https://github.com/commons-app/apps-android-commons/pull/5220/files#r1183342136).